x-pack/plugins/cloud_security_posture/common/constants.ts

```diff
@@ -36,4 +36,5 @@ export const INTERNAL_FEATURE_FLAGS = {
   showFindingsGroupBy: true,
 } as const;
 
-export const cspRuleAssetSavedObjectType = 'csp_rule';
+export const CSP_RULE_SAVED_OBJECT_TYPE = 'csp_rule';
```

Comment:
just noticed [other](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/server/features.ts#L124-L128) object types are registered with a dash, like `index-pattern` 

we might want to change this